components/raftstore/src/store/worker/cleanup_sst.rs

```diff
@@ -64,12 +66,37 @@ where
         }
     }
 
+    fn get_region_by_meta(&self, sst: &SstMeta) -> Result<Region> {
+        // The SST meta has been delivered with a range, use it directly.
```

Comment:
add a comment about BR passes the sst range while lightning doesn't.

Reply:
Both of br and lightning can trigger this situation in clean up ssts 🤔. In the normal case, the range in sst meta should not be empty in all cases.

Reply:
Yeah, this is just a simple guard for some "abnormal" cases, say, in the future someone changes the code...

Reply:
(Because `CleanupSstTask::ValidateSst` only has been constructed via the SST in `importer.list_ssts()`...)

Reply:
got it